---
title: "Publications"
---

```{r publications, echo=FALSE, message=FALSE, warnings=FALSE}
suppressWarnings(library(ggplot2))
suppressWarnings(library(plyr))
suppressWarnings(library(stringr))
suppressWarnings(library(scholar))
#### Create line graphs for N publications and citations over time ####
cite.history <- get_citation_history("rGDl8MAAAAAJ")
publications <- get_publications("rGDl8MAAAAAJ")
publications <- publications[
  !grepl("^[0-9]", publications$title) &
    !grepl("Rxiv|University", publications$journal) &
    !grepl("Supp|supp", publications$number),
]
publications$year[is.na(publications$year)] <- strsplit(date(), " ")[[1]][5]
publications <- ddply(publications, .(year), summarize, n = length(title))
for (i in cite.history$year) {
  if (!(i %in% publications$year)) {
    publications <- rbind(publications, c(i, 0))
  }
}
publications <- publications[order(publications$year), ]
plot.data <- data.frame(
  graph = c(
    rep("Cummulative number of publications", length(unique(publications$year))),
    rep("Cummulative number of Google Scholar reported citations", nrow(cite.history))
  ),
  x = as.numeric(c(publications$year, cite.history$year)),
  y = c(cumsum(publications$n), cumsum(cite.history$cites))
)
plot.data$graph <- factor(plot.data$graph, levels = unique(plot.data$graph))
g <- ggplot(plot.data, aes(x = x, y = y)) +
  geom_line() +
  geom_point(size = 2) +
  facet_grid(graph ~ ., switch = "y", scales = "free", labeller = label_wrap_gen(width = 18)) +
  scale_x_continuous(breaks = cite.history$year) +
  scale_y_continuous(breaks = scales::pretty_breaks(), position = "right") +
  theme_bw() +
  labs(x = "Year", y = "") +
  theme(
    axis.text.y = element_text(size = 10),
    axis.text.x = element_text(size = 10),
    strip.text.y = element_text(size = 10),
    plot.margin = margin(0.25, 0.25, 0, 0, "cm")
  )
ggsave("images/publications.png", g, width = 8, height = 2.8, device = "png")
```

![](images/publications.png){width=100%}

```{=html}
<head>
    <style>
        ul {
            list-style-type: decimal;
            padding: 10px;
        }
        li {
            margin: 20px 0;
        }
    </style>
</head>
<div>
    <ul>
        <li>
            <b>Wallen ZD</b>, Nesline MK, Pabla S, Gao S, Vanroey E, Hastings SB, Ko H, Strickland KC, Previs RA, Zhang S, Conroy JM, Jensen TJ, George E, Eisenberg M, Caveney B, Sathyan P, Ramkissoon S, Severson EA. A consensus-based classification workflow to determine genetically inferred ancestry from comprehensive genomic profiling of patients with solid tumors. <i>Brief Bioinform</i>. 25(6):bbae557 (2024). <a href="https://doi.org/10.1093/bib/bbae557">https://doi.org/10.1093/bib/bbae557</a>.
        </li>
        <li>
            Saini KS, Somara S, Ko HC, Thatai P, Quintana A, <b>Wallen ZD</b>, Green MF, Mehrotra R, McGuigan S, Pang L, Das S, Yadav K, Neric D, Cantini L, Joshi C, Iwamoto K, Dubbewar S, Vidal L, Chico I, Severson E, Lorini L, Badve S, Bossi P. Biomarkers in head and neck squamous cell carcinoma: unraveling the path to precision immunotherapy. <i>Front Oncol</i>. 14:1473706 (2024). <a href="https://doi.org/10.3389/fonc.2024.1473706">https://doi.org/10.3389/fonc.2024.1473706</a>.
        </li>
        <li>
            <b>Wallen ZD</b>, Ko H, Nesline MK, Hastings SB, Strickland KC, Previs RA, Zhang S, Pabla S, Conroy J, Jackson JB, Saini KS, Jensen TJ, Eisenberg M, Caveney B, Sathyan P, Severson EA, Ramkissoon SH. Real-world comprehensive genomic and immune profiling reveals distinct age-and sex-based genomic and immune landscapes in tumors of patients with non-small cell lung cancer. <i>Front Immunol</i>. 15:1413956 (2024). <a href="https://doi.org/10.3389/fimmu.2024.1413956">https://doi.org/10.3389/fimmu.2024.1413956</a>.
        </li>
        <li>
            Nesline MK, Subbiah V, Previs RA, Strickland KC, Ko H, DePietro P, Biorn MD, Cooper M, Wu N, Conroy J, Pabla S, Zhang S, <b>Wallen ZD</b>, Sathyan P, Saini K, Eisenberg M, Caveney B, Severson EA, Ramkissoon S. The Impact of Prior Single-Gene Testing on Comprehensive Genomic Profiling Results for Patients with Non-Small Cell Lung Cancer. <i>Oncol Ther</i>. 12(2):329-343 (2024). <a href="https://doi.org/10.1007/s40487-024-00270-x">https://doi.org/10.1007/s40487-024-00270-x</a>.
        </li>
        <li>
            Payami H, Cohen G, Murchison CF, Sampson TR, Standaert DG, <b>Wallen ZD</b>. Population fraction of Parkinson’s disease attributable to preventable risk factors. <i>npj Parkinsons Dis</i>. 9, 159 (2023). <a href="https://doi.org/10.1038/s41531-023-00603-z">https://doi.org/10.1038/s41531-023-00603-z</a>.
        </li>
        <li>
            <b>Wallen ZD</b>, Demirkan A, Twa G, Cohen G, Dean MN, Standaert DG, Sampson T, Payami H. Metagenomics of Parkinson's disease implicates the gut microbiome in multiple disease mechanisms. <i>Nat Comm</i>, 13, 6958 (2022). <a href="https://doi.org/10.1038/s41467-022-34667-x">https://doi.org/10.1038/s41467-022-34667-x</a>.
        </li>
        <li>
            Ilieva NM, <b>Wallen ZD</b>, De Miranda BR. Oral ingestion of the environmental toxicant trichlorethylene in rats induces alterations in the gut microbiome: Relevance to idiopathic Parkinson's disease. <i>Toxicology and Applied Pharmacology</i> 451 (2022). <a href="https://doi.org/10.1016/j.taap.2022.116176">https://doi.org/10.1016/j.taap.2022.116176</a>.
        </li>
        <li>
            <b>Wallen ZD</b>, Stone WJ, Factor SA, Molho E, Zabetian CP, Standaert DG, Payami H. Exploring human-genome gut-microbiome interaction in Parkinson's disease. <i>npj Parkinson's Dis</i> 7, 74 (2021). <a href="https://doi.org/10.1038/s41531-021-00218-2">https://doi.org/10.1038/s41531-021-00218-2</a>.
        </li>
        <li>
            <b>Wallen ZD</b>. Comparison study of differential abundance testing methods using two large Parkinson disease gut microbiome datasets derived from 16S amplicon sequencing. <i>BMC Bioinformatics</i> 22, 265 (2021). <a href="https://doi.org/10.1186/s12859-021-04193-6">https://doi.org/10.1186/s12859-021-04193-6</a>.
        </li>
        <li>
            Kurilshikov A, Medina-Gomez C, Bacigalupe R … <b>Wallen ZD</b> … Alexandra Zhernakova. Large-scale association analyses identify host factors influencing human gut microbiome composition. <i>Nat Genet</i> 53, 156-165 (2021). <a href="https://doi.org/10.1038/s41588-020-00763-1">https://doi.org/10.1038/s41588-020-00763-1</a>.
        </li>
        <li>
            <b>Wallen ZD</b>, Appah M, Dean MN, Sesler CL, Factor SA, Molho E, Zabetian CP, Standaert DG, Payami H. Characterizing dysbiosis of gut microbiome in PD: evidence for overabundance of opportunistic pathogens. <i>npj Parkinson's Dis</i>. 2020 Jun 12;6:11. <a href="https://doi.org/10.1038/s41531-020-0112-6">https://doi.org/10.1038/s41531-020-0112-6</a>.
        </li>
        <li>
            Schonhoff AM, Williams GP, <b>Wallen ZD</b>, Standaert DG, Harms AS. Innate and adaptive immune responses in Parkinson's disease. <i>Prog Brain Res</i>. 2020;252:169-216. <a href="https://doi.org/10.1016/bs.pbr.2019.10.006">https://doi.org/10.1016/bs.pbr.2019.10.006</a>.
        </li>
        <li>
            <b>Wallen ZD</b>, Chen H, Hill-Burns EM, Factor SA, Zabetian CP, Payami H. Plasticity-related gene 3 and age at diagnosis of Parkinson disease. <i>Neurol Genet</i>. 2018 Oct; 4(5): e271. <a href="https://doi.org/10.1212/NXG.0000000000000271">https://doi.org/10.1212/NXG.0000000000000271</a>.
        </li>
        <li>
            Wang J, Kurilshikov A, Radjabzadeh D, Turpin W, Croitoru K, Bonder MJ, Jackson MA, Medina-Gomez C, Frost F, Homuth G, Rühlemann M, Hughes D, Kim HN; MiBioGen Consortium Initiative, Spector TD, Bell JT, Steves CJ, Timpson N, Franke A, Wijmenga C, Meyer K, Kacprowski T, Franke L, Paterson AD, Raes J, Kraaij R, Zhernakova A. Meta-analysis of human genome-microbiome association studies: the MiBioGen consortium initiative. <i>Microbiome</i>, 2018 Jun 8;6(1):101. <b>[Listed as collaborator]</b> <a href="https://doi.org/10.1186/s40168-018-0479-3">https://doi.org/10.1186/s40168-018-0479-3</a>.
        </li>
        <li>
            Hill-Burns EM, Debelius JW, Morton JT, Wissemann WT, Lewis MR, <b>Wallen ZD</b>, Peddada SD, Factor SA, Molho E, Zabetian CP, Knight R, Payami H. Parkinson's disease and Parkinson's disease medications have distinct signatures of the gut microbiome. <i>Mov Disord</i>, 32: 739-749. <a href="https://doi.org/10.1002/mds.26942">https://doi.org/10.1002/mds.26942</a>.
        </li>
    </ul>
</div>
```